Microsoft/Recognizers-Text
# [NL NumberWithUnit] Add support for Age, Dimension and Temperature .NET

_Depends on PR #795, please merge that one first_

- Add support for Age (16 testcases OK)
- Add support for Temperature (3 testcases skipped, 29 testcases OK)
- Add support for Dimension (2 testcases skipped (see #800), 58 testcases OK)
- Fix faulty testcases
- Added more testcases for DimensionModel since many possibilities were not covered yet
